codespell_lib/data/dictionary.txt

```diff
@@ -3057,8 +3096,11 @@ cliuent->client
 cliuents->clients
 cloack->cloak
 cloacks->cloaks
+cloberring->clobbering
 clockwíse->clockwise
 clodes->closed, clothes,
+cloneable->clonable
+clonez->cloner
```

Comment:
What about:
```
clonez->cloner, clones,
```

Thinking about it, I am not sure if `cloner` is valid

Reply:
It's common to find it in code.

Anyway, added.